pkg/api/persistentvolumeclaim/util_test.go

```diff
@@ -167,36 +167,15 @@ func TestPVCDataSourceSpecFilter(t *testing.T) {
 			gateEnabled: true,
 			want:        validSpec.DataSource,
 		},
-		"disabled with invalid spec": {
-			spec:        invalidSpec,
-			gateEnabled: false,
```

Comment:
I should have spotted it earlier: `gateEnabled` is now useless and should be removed from the struct.